scripts/generate-app-config/generateAppConfig.js

```diff
@@ -61,49 +66,64 @@ const generateAppConfig = (siteConfig, production, verbose) => {
     imports,
   );
 
-  const routingConfig = addConfig(
-    generateRouteConfig(siteConfig, generatePagesConfig(siteConfig, production, verbose)),
-    'routeConfig.js',
+  const contentConfig = generateContentConfig(siteConfig, generatePagesConfig(siteConfig, production, verbose));
```

Comment:
nit: might be cleaner to do `cont { content, menuItems} = generate....` like you when pulling the generated nav items config